Neos.Flow/Classes/I18n/Xliff/Service/XliffFileProvider.php

```diff
@@ -87,26 +94,30 @@ public function initializeObject()
      */
     public function getMergedFileData($fileId, Locale $locale): array
     {
-        if (!isset($this->files[$fileId][$locale->getLanguage()])) {
```

Comment:
hmm feels wrong to me to rely on the magic `__toString()` method, maybe add a `getIdentifier` or `getLocaleIdentifier` method and use that instead? `__toString()` could do that same then of course..

but maybe that's just me..

Reply:
I'd say a `getIdentifier()` method in `\Neos\Flow\I18n\Locale` is generally a good thing, but it's also a medium change to Flow if you consequently replace each `Locale` string cast with the new method. This is just a humble bugfix ;)

Reply:
you're right, seems like it's used a bunch of other places in the code so guess someone meant to use that way.. nvm then